static/self-hosted/aks/config.yaml

```diff
@@ -0,0 +1,13 @@
+license: 
+
+subdomain: 
```

Comment:
@codyjlandstrom why are we creating these new static files as part of the docusaurus upgrade?